src/ResourceManager/Consumption/Commands.Consumption/Models/PSUsageDetail.cs

```diff
@@ -50,25 +60,34 @@ public PSUsageDetail(ApiUsageDetail usageDetail)
         {
             if (usageDetail != null)
             {
-                this.Id = usageDetail.Id;
-                this.Type = usageDetail.Type;
-                this.Name = usageDetail.Name;
-                this.Tags = usageDetail.Tags;
-                this.UsageStart = usageDetail.UsageStart;
-                this.UsageEnd = usageDetail.UsageEnd;
+                this.AccountName = usageDetail.AccountName;
+                this.AdditionalInfo = usageDetail.AdditionalProperties;
```

Comment:
It will always be null. AdditionalProperties of dictionary type was developed by previous team long ago. We don't have information on how to populate it in a dictionary type. Instead we do AdditionInfo in string type, and mark AdditionalProperties as future breaking change. Let me know your advice in that case.

Reply:
As long as the current behavior is that it always returns null, it is fine that it now will not be populated.